pom.xml

```diff
@@ -52,7 +52,7 @@
                     </executions>
                     <configuration>
                         <repository>${docker.repository}</repository>
-                        <tag>v${project.version}</tag>
+                        <tag>${project.version}</tag>
```

Comment:
Is this docker image tag?

Reply:
wso2/k8s-api-operator                                                      2.0.2            90262d19c75e   40 hours ago    136MB

Yep this will push the image I guess the latest tag is in the same format
https://hub.docker.com/layers/k8s-api-operator/wso2/k8s-api-operator/2.0.2/images/sha256-869618ca6288b59032a7331093b6cd965ee0356d735e7f87793e2ec8dd7cb786?context=explore

Reply:
This is used only when mvn dockerfile:push